[INFO -- 2017-03-08 00:01:51]
Starting flip_value script
version: 2.0
[INFO -- 2017-03-08 00:01:51]
Initialization strings:
[INFO -- 2017-03-08 00:01:51]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-03-08 00:01:51]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-08 00:01:51]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-08 00:01:51]
(4) file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-08 00:01:51]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-08 00:01:51]
(6) set args 228 2048 16 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_a_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_b_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/gold_2048_m-order_228_ths_16_blocks 1
[INFO -- 2017-03-08 00:01:51]
initSignal:21
[INFO -- 2017-03-08 00:01:51]
endSignal:28
[INFO -- 2017-03-08 00:01:51]
seqSignal:15
[INFO -- 2017-03-08 00:01:51]
faultMode:2
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:02:25]
pythonException: invalid frame
[INFO -- 2017-03-08 00:02:25]
Backtrace BEGIN:
#0  0x000000000040372f in L_dgemm_185__par_region0_2_0 () at dgemm.c:221
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffe550 in ?? ()
#3  0x0000000000000000 in ?? ()

216	                            CC_arr(i,j) = 0.0;
217	
218	                    for (kg=kk,k=0; kg<MIN(kk+block,order); k++,kg++)
219	                        for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
220	                            for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
221	                                CC_arr(i,j) += AA_arr(i,k)*BB_arr(j,k);
222	
223	                    for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
224	                        for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
225	                            C_arr(ig,jg) += CC_arr(i,j);

Backtrace END
Memory content before bitflip:0x0a0x000x000x00
Fault Mode: Random bit-flip
base address to flip value: 0x7fffffffdeec
address max offset: 4
Memory content after  bitflip:0xb10xa80xa70x64
frame name: L_dgemm_185__par_region0_2_0
symbol name: i
symbol filename: dgemm.c
symbol line: 183
value: 1688709297
value address: 0x7fffffffdeec
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2017-03-08 00:02:25]
Fault Injection Successful after 34.8557128906s
[INFO -- 2017-03-08 00:02:26]
event type: exit
[EXCEPTION -- 2017-03-08 00:02:26]
exit code: no exit code available
